packages/react-app/src/pages/About/index.tsx

```diff
@@ -48,6 +48,44 @@ export const About: React.FC = () => (
           ></img>
         </a>
       </div>
+      <div className={styles.bottom_bar}>
+        <div className={styles.outro}>
+          <h1 className={styles.lower_landing_header}>Attributions</h1>
+          <p className={styles.lower_landing_subtitle}>
+            Climate and terrain data used to generate these maps
+          </p>
+          <p className={styles.paragraph}>
```

Comment:
Let's add a subtitle for each of these, just couple words explaining what the data is